### PR TITLE
feat: added QR Code including service address

### PR DESCRIPTION
**Pull request by maudnals**
## Fixes KILTProtocol/ticket#316
* Added a QR code displaying the public identity including a `serviceAddress` to the identity card (visible under `/dashboard`).
* Added a minimal "encoding" setup (simply an array), for the QR Code to be readable by mobile cameras.

## How to test:

- Checkout the branch and run locally alongside the node and the services
- Create a new identity e.g. with the mnemonic "dog hen effort cradle win embark tuition need fly stumble short damage"
- You should see a field Public Identity as below. By default the QR Code should be hidden. 
- Click the button "Show". The QR Code should be displayed and the button label verb should be "Hide".
- Click the button again to hide.

Screenshot 2020-02-18 at 12 08 32
Screenshot 2020-02-18 at 12 08 38

- Scan the QR Code with a mobile phone
- Your phone should read the QR Code and display the following data:
`["5EwY7eCYAAcLwANHv8s18m5LnYnPyJM8PhvaWWqkeJq4QVtu","0xdbcea99a89c26039f6b05d8d458fafa308524ca1382b7b5f8a3b6482a871a864","//127.0.0.1:3000/messaging"]`

IMG_0416
IMG_0417

NB: follow-up tickets: 
* https://github.com/KILTprotocol/ticket/issues/317
* https://github.com/KILTprotocol/ticket/issues/318

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

File: src/assets/kilt_small.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tjwelde) [inline src/assets/kilt_small.svg]: This can be optimized

File: src/utils/PublicIdentity/Encoding.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  a: string
+  b: string
+  s?: string
+}
+
+const encodePublicIdentity = (
+  publicIdentity: IPublicIdentity
+): TPublicIdentityEncoded => ({
+  a: publicIdentity.address,
+  b: publicIdentity.boxPublicKeyAsHex,
```
Comment (tjwelde) [inline src/utils/PublicIdentity/Encoding.ts]: If you want to have it even smaller, you could convert it to base64 first

File: src/utils/PublicIdentity/Encoding.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+type TPublicIdentityEncoded = {
+  a: string
+  b: string
+  s?: string
+}
+
+const encodePublicIdentity = (
+  publicIdentity: IPublicIdentity
+): TPublicIdentityEncoded => ({
+  a: publicIdentity.address,
```
Comment (tjwelde) [inline src/utils/PublicIdentity/Encoding.ts]: you could even have an array here

File: src/assets/kilt_small.svg (hunk 1 above)
Comment (maudnals) [reply src/assets/kilt_small.svg]: Right, thanks, fixed via this https://github.com/svg/svgo 👍

File: src/utils/PublicIdentity/Encoding.ts (hunk 3 above)
Comment (maudnals) [reply src/utils/PublicIdentity/Encoding.ts]: Yes that's better, done

File: src/utils/PublicIdentity/Encoding.ts (hunk 2 above)
Comment (maudnals) [reply src/utils/PublicIdentity/Encoding.ts]: Good idea; for now I'm sticking to a simple array.

**Pull request merged**